modules/edu.gemini.epics.acm/src/main/java/edu/gemini/epics/acm/CaObserveSenderImpl.java

```diff
@@ -829,6 +829,21 @@ private void failCommandWithCarError(final CaCommandMonitorImpl cm) {
         } );
     }
 
+    private void failCommandWithObserveCarError(final CaCommandMonitorImpl cm) {
+        // I found that if I try to read OMSS or MESS from the same thread that
```

Comment:
Do you thik that's happening at CAS level?

Reply:
I am not sure. This comment is from like 3 years ago.